frontcamp-webpack/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "frontcamp-webpack",
+  "version": "1.0.0",
+  "description": "FrontCamp Webpack",
+  "main": "index.js",
+  "author": "Aliaksandr Khvainitski",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/driv3r23/frontcamp.git"
+  },
+  "scripts": {
+    "publish": "webpack --config webpack.production.config.js",
+    "build": "webpack --config webpack.development.config.js",
+    "server": "webpack-dev-server",
+    "start": "npm-run-all --parallel build server",
```

Comment:
It actually has no much sense.  
I expected to have possibility to run your application using only webpack-dev-server, without generation files for it using webpack. 
Webpack-devserver - great tool, and I would like you to try it, it's really cool for development, because different features such as watch and etc.

Please try to fix your configs to have possibility to run application using webpack-dev-server.